src/OrbitGl/PrimitiveAssembler.cpp

```diff
@@ -191,7 +194,7 @@ void PrimitiveAssembler::AddTriangle(const Triangle& triangle, const Color& colo
                                      std::shared_ptr<Pickable> pickable) {
   ORBIT_CHECK(picking_manager_ != nullptr);
 
-  Color picking_color = picking_manager_->GetPickableColor(pickable, batcher_id_);
+  Color picking_color = GetPickingManager()->GetPickableColor(pickable, GetBatcherId());
```

Comment:
I don't understand this change. Aren't `GetPickingManager` and `GetBatcherId` just returning the local variables? Am I missing something?

Reply:
BatcherId is not a local variable anymore, but I will revert PickingManager one, this was going to be moved to Batcher but I changed my mind.